### DISCUSSION TITLE
[Samples] Return to case after deleting a sample from the case view

### DISCUSSION THREAD

**Discussion by AllieLerbs** [SORMAS-Foundation/SORMAS-Project#12009] [Issue Discussions]
### Problem Description

When you are in a case and click on the pen-button in the samples section the system change to samples and display the simple sample. In this overview it is possible to delete the sample. After deleting the sample the system remains in the sample directory

### Proposed Change

After deleting a sample from case view set trigger to return to the case from the deleted sample.

### Added Value/Benefit

For a better traceability and case management the user can work on the same case without have to search again for the case.

### Acceptance Criteria

- [ ]
- [ ]
- [ ]

### Implementation Details
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/91321821/ea1127f0-4128-495f-b7fc-c7922e62c04e)

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/91321821/8027ea45-9cfe-4ca1-b0bf-feca642267b0)

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/91321821/138e88b5-ed50-4a6d-9b8f-0c4b028f37ea)

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/91321821/0f9f5cb8-9182-4674-90b1-0f7fea038691)

**Comment by MateStrysewske** [comment]
@AliciaRaber Where's this request coming from? I find it a bit strange in terms of workflow that if I enter a sample through the sample directory, and I delete it, I suddenly get redirected back to a case. Note that clicking on "See samples for this person" brings up all samples related to that person, not only the case in question. Those samples are already shown in the side component. From my perspective, if a user clicks on "See samples for this person" and takes action on those samples, they've already left the scope of the case.

I definitely see this applied to a slightly different workflow though; because, if I directly enter a sample through the case (i.e. not clicking on "See samples for this person", but on the edit icon of a specific sample) and delete it from there, I also get redirected to the samples directory; here I would agree that it's better for workflow if the user gets taken back to the case instead.

**Comment by AllieLerbs** [comment]
Hello Maté. The request was opened by a health department (Munich). Would be great if after deleting a sample the UI redirect to the case.
@MateStrysewske

**Comment by MateStrysewske** [comment]
@AliciaRaber Thanks!

Converted this issue into a discussion because I think we need to decide whether we agree with this change request or not.

**Comment by AllieLerbs** [comment]
@MateStrysewske
As discussed I have updated the screenshots an refine the description of the behavior. Please let me know if you need further information.

**Comment by AllieLerbs** [comment]
https://github.com/SORMAS-Foundation/SORMAS-Project/issues/11861
Original issue
